cypress/e2e/cloud/releases.test.ts

```diff
@@ -0,0 +1,148 @@
+import {Organization} from '../../../src/types'
+
+// webpack bundling error if try importing --> due to imports in the constants file
+const IOX_SWITCHOVER_CREATION_DATE = '2023-01-31T00:00:00Z'
```

Comment:
FYI -- why not importing this constant.
<img width="471" alt="Screen Shot 2023-02-01 at 10 37 25 PM" src="https://user-images.githubusercontent.com/10232835/216250335-d0798f03-dbd7-4da4-b790-e8d28fc9e1b2.png">